net/net.go

```diff
@@ -1022,37 +1032,22 @@ func (n *net) startPulling() {
 	}
 }
 
-// getOwnLoad returns the log owned by the host under the given thread.
-func (n *net) getOwnLog(id thread.ID) (info thread.LogInfo, err error) {
-	logs, err := n.store.LogsWithKeys(id)
-	if err != nil {
-		return
-	}
-	for _, lid := range logs {
-		sk, err := n.store.PrivKey(id, lid)
-		if err != nil {
-			return info, err
-		}
-		if sk != nil {
-			return n.store.GetLog(id, lid)
-		}
-	}
-	return info, nil
-}
-
-// getOrCreateOwnLoad returns the log owned by the host under the given thread.
-// If no log exists, a new one is created under the given thread.
+// getOrCreateOwnLoad returns the first log 'owned' by the host under the given thread.
+// If no log exists, a new one is created under the given thread and returned.
+// This is a strict 'owership' check, vs returning the first directly 'managed' log.
 func (n *net) getOrCreateOwnLog(id thread.ID) (info thread.LogInfo, err error) {
-	info, err = n.getOwnLog(id)
+	lgs, err := n.store.GetOwnLogs(id)
 	if err != nil {
 		return info, err
 	}
-	if info.PubKey != nil {
```

Comment:
Is this right? PubKey here? Seems like it should have been PrivKey (see update), but its possible I'm missing something and PubKey is sufficient.

Reply:
`getOwnLog` used to return an empty struct if own log wasn't found, so a check for pubkey check if it's _really_ a log or not.

Reply:
Does `getOwnLog` always return an inflated struct now?

Reply:
Yes, and it errors now with NotFound if it doesn't find any logs. So this additional check is now redundant. Will update.